osu.Game/Overlays/Settings/Sections/DebugSettings/MemorySettings.cs

```diff
@@ -33,6 +33,10 @@ private void load(GameHost host, RealmContextFactory realmFactory)
                         using (realmFactory.BlockAllOperations())
                         {
                         }
+
+                        // retrieve context to revive realm subscriptions.
+                        // TODO: should we do this from OsuGame or RealmContextFactory or something? Answer: yes.
+                        var _ = realmFactory.Context;
```

Comment:
Still have to move this one. Not sure on the best mechanism for handling this just yet. Potentially a post to `SynchronizationContext`.